### ISSUE TITLE
The model does not converge based on the loss curve.

### ISSUE THREAD

**Issue by liuyvchi** [yihongXU/deepMOT#41]
I meet the same issue as issue #40. The trained model works well according to the evaluation results. However, the training loss does not decrease during training just like issue #40
